test/Microsoft.Extensions.Logging.Test/ConsoleLoggerTest.cs

```diff
@@ -649,7 +650,7 @@ public void CallingBeginScopeOnLogger_ReturnsNonNullableInstance()
             Assert.NotNull(disposable);
         }
 
-#if NET452
+#if NET46
```

Comment:
This test shouldn't be needed anymore now that we are using `AsyncLocal` in .NET Framework